src/sentry/runner/commands/run.py

```diff
@@ -283,6 +283,11 @@ def taskworker_scheduler(redis_cluster: str, **options: Any) -> None:
     "--max-task-count", help="Number of tasks this worker should run before exiting", default=10000
 )
 @click.option("--concurrency", help="Number of child worker processes to create.", default=1)
+@click.option(
+    "--prefetch-multiplier",
+    help="How many tasks to keep in the child worker Queue. Multiplied by --concurrency",
+    default=3,
```

Comment:
Why such a low default?

Reply:
I was trying to minimize the number of activations that are 'aging' in a worker, and try to minimize empty queues. Currently our celery workers have a prefetch of 4.